src/graphic/wordwrap.cc

```diff
@@ -367,11 +370,9 @@ void WordWrap::draw(RenderTarget& dst,
 	calc_wrapped_pos(selection_start, selection_start_line, selection_start_x);
 	calc_wrapped_pos(selection_end, selection_end_line, selection_end_x);
 
-	++where.y;
-
```

Comment:
Not sure, whether this is the place, but the text highlighting is now moved up a bit and is no longer vertically centered.